packages/help-center/src/components/utils.tsx

```diff
@@ -99,3 +99,15 @@ export const calculateUnread = ( conversations: ZendeskConversation[] ) => {
 
 	return { unreadConversations, unreadMessages };
 };
+
+export const getClientId = ( conversations: ZendeskConversation[] ) => {
```

Comment:
I think we could reduce the nested loop to make it faster/simpler. I have tested and works great.

```js
export const getClientId = ( conversations: ZendeskConversation[] ): string =>
	conversations
		.flatMap( ( conversation ) => conversation.messages )
		.find( ( message ) => message.source.type === 'web' && message.source.id )?.source.id || '';
```